fastddsspy_participants/src/cpp/model/DataStreamer.cpp

```diff
@@ -142,6 +144,12 @@ void DataStreamer::add_data(
         {
             dyn_type = it->second;
         }
+
+        auto map_it = topic_type_discovered_.find(topic.m_topic_name);
```

Comment:
Comment for line 136: watch out, this could actually occur now right? You could print topic_* , be able to activate the streamer (since for example topic_A exists and its type is discovered) and then receive a message of topic_B whose type is not discovered.

Reply:
Better know with a warning instead, but please also remove the code comment.